indexd/default_settings.py

```diff
@@ -17,6 +17,8 @@
         'sqlite:///alias.sq3', auto_migrate=AUTO_MIGRATE),
 }
 
+CONFIG['DIST'] = []
```

Comment:
can we rename the dist variable(also blueprint.dist) to be more descriptive? I just don't understand what's suppose to go in here...